### ISSUE TITLE
Add average speed enforcement camera support

### ISSUE THREAD

**Issue by Talkless** [osmandapp/OsmAnd#13430]
# 🚀 feature request

### Description

Add notifications (on-screen and voice) and map icons for average speed cameras. In Lithuania, we have increasing number of such average speed sections:

https://overpass-turbo.eu/s/1eqE

In Lithuanian these average speed sections are available from public government sources, and I've personally added quite some of these average speed sections recently.

### Describe the solution you'd like

Average speed sections are defines as `enforcement=average_speed` relation, with `from`, `to` nodes, two `device` nodes, and set of `section`'s:
https://wiki.openstreetmap.org/wiki/Relation:enforcement#Members

Probably average speed camera sign should be visible whole time while traversing enforced sections, until `to` node is reached. If average speed is measured in both ways, there are two relations with same `device`'es, `section`'s, but with`from` and `to` swapped.

I am not sure how on-screen sign should look like to distinguish from immediate speed camera. Duckduck image search suggest same speed camera sign with "average speed" text added:

https://duckduckgo.com/?q=average+speed+cameras+sign&atb=v35-1&iar=images&iax=images&ia=images

No idea who speed camera should be visible on map, should it have different icon to the current immediate speed camera or different..?

**Comment by alex9099** [comment]
This would be nice to see implemented. 

In portugal we use https://www.anecrarevista.pt/wp-content/uploads/2021/03/velocidade-Media_Anecra_Revista-1320x1916.jpg

I don't think there's any norm in europe (let alone worldwide)

**Comment by Talkless** [comment]
This is how it looks in Lithuanian: a speed camera sign with range specified below:
![external-content duckduckgo com](https://user-images.githubusercontent.com/292191/147456948-2587b31d-a0cf-417c-bd85-e9f40bd277a2.jpeg)

**Comment by Talkless** [comment]
Looks like `segment`'s enforcement relation memers are not necessary:
> Line for the section control (SPECS / Section Control), if not defined by the from and to roles above. 

"**if not** defined by".
From:
https://wiki.openstreetmap.org/wiki/Relation:enforcement#Members

This is example with only from/to/device: https://www.openstreetmap.org/relation/1737016

**Comment by bobeeeze** [comment]
It would be really nice if Osmand would add the notification for the average speed cameras because there are more and more of them (in Europe at least) and for the moment Osmand does not even warn of the entry in the section. And it would be nice that the speed camera icon flashes all along the section until you leave the controlled section. I hope that the Osmand team will react on this subject because it's been more than a year since this issue was opened.
A big thank you for this great application

**Comment by ireun** [comment]
Bumping on this one, there is more and more avg. speed cameras, which does not seem to be picked up by OsmAnd

**Comment by linuxthrawn** [comment]
Yep, more and more countries are adding such average speed segments. It would be nice to have a way to detect, notify and visualize in OSMAND. This is the current list in Bulgaria.

<img width="771" height="913" alt="Image" src="https://github.com/user-attachments/assets/0e739e7a-8e14-4be3-b7b5-6cf2aa4a621b" />

This is the official site: https://www.bgtoll.bg/en/questions-answers .
